content/webapp/components/EventSchedule/EventScheduleItem.tsx

```diff
@@ -98,30 +99,58 @@ const eventLocations = (locations: Place[], isHybridEvent: boolean) => {
 
 // We have a message block on scheduled events which either displays
```

Comment:
```suggestion
// We have a message block on scheduled events which can display either
```

mega np adding the "can" because it doesn't necessarily display anything.